transmit-lib/src/cmr/transmit/access_control.clj

```diff
@@ -201,6 +202,13 @@
   ([context params options]
    (search-for-acls* context params (merge options {:method :post}))))
 
+(defn search-for-acls-with-returned-headers
```

Comment:
Why can't we just use search-for-acls and put the header information in the downstream functions?  I don't think we need to duplicate all of the functions 1 to handle the exact same requests with headers and one without. We should always handle the headers.

Reply:
because 1. it causes a lot of test failures, 2. if you look at the detail of the function, there is some differences other than returning headers.